_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -33,5 +30,45 @@ <h3>Snap Support</h3>
     remove certbot</tt>, or <tt>sudo yum remove certbot</tt>.
     </p>
 </li>
+
+
+{{#advanced}}
+{{#dns_plugins}}
+{{>installcertbotwildcard}}
+<li>
+  Confirm plugin containment level
+  <p>
+  Run this command on the command line on the machine to acknowledge that the installed
+  plugin will have the same <tt>classic</tt> containment as the Certbot snap.
+  </p>
+  <pre>sudo snap set certbot trust-plugin-with-root=ok</pre>
+
+  <p>
+  If you encounter issues with running Certbot, you may need to follow this step, then
+  the "Install correct DNS plugin" step, again.
+  </p>
+</li>
+{{/dns_plugins}}
+{{/advanced}}
+
+{{^advanced}}
 {{>installcertbot}}
+{{/advanced}}
+
 {{> dnspluginssetup}}
+
+{{#advanced}}
+{{#dns_plugins}}
+<li>
+  Connect the DNS plugin snap to the Certbot snap
+  <p>
+    Run the following command, replacing &lt;PLUGIN&gt; with the name of your DNS provider.
+  </p>
+  <pre>sudo snap connect certbot:plugin {{dns_package_prefix}}-&lt;PLUGIN&gt;</pre>
+  <p>
+    For example, if your DNS provider is Cloudflare, you'd run the following command:
+  </p>
+  <pre>sudo snap connect certbot:plugin {{dns_package_prefix}}-cloudflare</pre>
```

Comment:
This and the more general example above generates commands like:
```
sudo snap connect certbot:plugin --beta certbot-dns-cloudflare
```
which for me results in:
```
error: unknown flag `beta'
```

Reply:
the solution I went with isn't the cleanest thing I've ever written, but I think it's better than the alternatives, which I saw as:
- hard-code the prefix of the plugin in `snap.html`
- refactor things again to deal with `--beta` being required for certbot install only on the wildcard tab
- hard-code install instructions for snaps

Reply:
Especially since this should just be temporary until we have stable DNS plugin snaps, I agree it's OK.